streams/cluster/src/main/java/com/hortonworks/streamline/streams/cluster/register/impl/AbstractServiceRegistrar.java

```diff
@@ -83,7 +85,7 @@ public Service register(Cluster cluster, Config config, List<ConfigFileInfo> con
             throw new IllegalArgumentException("Validation failed for service configurations.");
         }
 
-        if (!validateServiceConfiguationsViaFlattenMap(flattenConfigMap)) {
+        if (!validateServiceConfiguationsAsFlattenedMap(flattenConfigMap)) {
```

Comment:
nit: perhaps a better name would be: `isValidServiceConfiguationsAsFlattenedMap(...)`